R/CreatePackageReport.R

```diff
@@ -39,14 +43,21 @@ CreatePackageReport <- function(packageName
                    , paste(unlist(lapply(packageReporters, function(x) class(x)[1]))
                            , collapse = "\n")))
 
+
     # Make them plots
     for (reporter in packageReporters){
         log_info(paste("Running Package Reporter", class(reporter)[1]))
         reporter$set_package(packageName, packagePath)
         
         reporter$calculate_all_metrics()
-        # TODO: replace plot_network() with render_report() which is then rendered into a parent report.
-        reporter$plot_network()
+        
+        
+        # For AbstractGraphReporter Types Only
+        if ("AbstractGraphReporter" %in% class(reporter)) {
```

Comment:
per my comment above....push the complexity of this `orphanNodeClusteringThreshold` business into the class constructors and keep this function clean